lib/pages/speakers_page.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:convert';
+
+import 'package:intl/intl.dart';
+import 'package:jaspr/components.dart';
+import 'package:jaspr/html.dart';
+import 'package:jaspr_router/jaspr_router.dart';
+import 'package:http/http.dart' as http;
+
+// class AgendaPage extends StatelessComponent {
```

Comment:
This is probably copy/paste error but I assume this will need to be SpeakerPage and that we will uncomment it when we are ready to display?